### PR TITLE
Add to installation requirements

### PR DESCRIPTION
**Pull request by dadlerj**

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -7,12 +7,16 @@
...
+- [Kubernetes](https://kubernetes.io/) v1.9 or later with an SSD storage class
+  - Cluster role administrator access
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) v1.9.7 or later
 - Access to server infrastructure on which you can create a Kubernetes cluster (see
   [resource allocation guidelines](scale.md)).
 - [Sourcegraph Enterprise license](./configure.md#add-license-key). You can run through these instructions without one, but you must obtain a license for instances of more than 10 users.
-
+- A domain name for your Sourcegraph instance ([to enable SSL/TLS](https://github.com/sourcegraph/deploy-sourcegraph/blob/master/docs/configure.md#configure-tlsssl))
+- A certificate (self-signed or from a service such as [Let's Encrypt](https://letsencrypt.org/))
+- Access tokens or other credentials to connect to your code hosts of choice
```
Comment (dadlerj) [inline docs/install.md:18]:
```suggestion
- Access tokens or other credentials to [connect to your code hosts of choice](https://docs.sourcegraph.com/admin/external_service)
```

File: docs/install.md [hunk 2]
```diff
@@ -7,12 +7,16 @@
...
-- [Kubernetes](https://kubernetes.io/) v1.9 or later
+- [Kubernetes](https://kubernetes.io/) v1.9 or later with an SSD storage class
+  - Cluster role administrator access
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) v1.9.7 or later
 - Access to server infrastructure on which you can create a Kubernetes cluster (see
   [resource allocation guidelines](scale.md)).
 - [Sourcegraph Enterprise license](./configure.md#add-license-key). You can run through these instructions without one, but you must obtain a license for instances of more than 10 users.
-
+- A domain name for your Sourcegraph instance ([to enable SSL/TLS](https://github.com/sourcegraph/deploy-sourcegraph/blob/master/docs/configure.md#configure-tlsssl))
+- A certificate (self-signed or from a service such as [Let's Encrypt](https://letsencrypt.org/))
```
Comment (ggilmore) [inline docs/install.md:17]: There are a number of options here:

1) A valid certificate issued by a trusted certificate authority (such as Comodo, RapidSSL, etc.)
2) A self-signed certificate that you can distribute and install across all of your user's machines
3) An existing reverse proxy that provides SSL termination for the (such as Cloudflare)

File: docs/install.md [hunk 3]
```diff
@@ -7,12 +7,16 @@
 
 ## Requirements
 
-- [Kubernetes](https://kubernetes.io/) v1.9 or later
+- [Kubernetes](https://kubernetes.io/) v1.9 or later with an SSD storage class
+  - Cluster role administrator access
```
Comment (ggilmore) [inline docs/install.md:11]: I would link to https://kubernetes.io/docs/reference/access-authn-authz/rbac/ here (I can't directly link to `cluster-admin`, but there are a number of references to it on this page)

Comment (dadlerj) [inline docs/install.md:11]:
```suggestion
  - [Cluster role administrator access](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)
```

File: docs/install.md (hunk 2 above)
Comment (dadlerj) [reply docs/install.md:17]: What's a good way to describe this at a high level? E.g. "A way to ___" or "A method for ___" or "A certificate or ___". 

I.e., what's the header bullet if this list were sub-bullets?

Comment (uwedeportivo) [reply docs/install.md:17]: i would say: "A TLS certificate"

File: docs/install.md [hunk 4]
```diff
@@ -7,12 +7,16 @@
...
-- [Kubernetes](https://kubernetes.io/) v1.9 or later
+- [Kubernetes](https://kubernetes.io/) v1.9 or later with an SSD storage class
+  - [Cluster role administrator access](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) v1.9.7 or later
 - Access to server infrastructure on which you can create a Kubernetes cluster (see
   [resource allocation guidelines](scale.md)).
 - [Sourcegraph Enterprise license](./configure.md#add-license-key). You can run through these instructions without one, but you must obtain a license for instances of more than 10 users.
-
+- A valid domain name for your Sourcegraph instance ([to enable SSL/TLS](https://github.com/sourcegraph/deploy-sourcegraph/blob/master/docs/configure.md#configure-tlsssl))
+- A certificate (self-signed or from a service such as [Let's Encrypt](https://letsencrypt.org/))
```
Comment (dadlerj) [inline docs/install.md:17]:
```suggestion
- A valid TLS certificate (whether from a trusted certificate authority such as Comodo, RapidSSL, or others, a self-signed certificate that can be distributed and installed across all users' machines, or the ability to use an existing reverse proxy that provides SSL termination for the connection)
```

**Pull request merged**